woocommerce/woocommerce-ios
# Migrate Observables to Combine for a few classes

Parts of #4379 


### Description


We used to have our own implementation of observables before we can use Combine in iOS 13, and now we're ready to migrate to Combine in the codebase. This PR contains the changes for a few classes:

- `ScrollWatcher` - subscribed in `PaginatedListSelectorViewController`
- `StoreStatsAndTopPerformersPeriodViewModel` - subscribed in `OldStoreStatsAndTopPerformersPeriodViewController` and `StoreStatsAndTopPerformersPeriodViewController`
- `ProductInventorySettingsViewModel` - subscribed in `ProductInventorySettingsViewController`
- `FilterListViewController.StaticListSelectorCommand` - subscribed in `FilterListViewController`
- `TitleAndEditableValueTableViewCellViewModel` - subscribed in `ManualTrackingViewController`

### Testing instructions


Please feel free to do a confidence check on any of the following screens:

- Add/Edit Product > Shipping > Shipping class
- Add/Edit Product > Price > Tax class
- Add/Edit Product > Inventory --> the sections should be shown as before
- My store tab: in-app feedback card visibility
- Orders tab > Filter
- Products tab > Filter
- Order details > Add tracking


---
